content/en/blog/2019/monitoring-external-service-traffic/index.md

```diff
@@ -98,7 +98,7 @@ This is where the BlackHole and Passthrough clusters are used.
     }
   {{< /text >}}
 
-  The route is setup as [direct response](https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/route/route_components.proto#envoy-api-field-route-route-direct-response)
+  The route is setup as [direct response](https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/route/route_components.proto#envoy-api-msg-route-directresponseaction)
```

Comment:
Also, why are you changing an en file. You should not be changing content files from other languages.

Reply:
I did not make the change, it was the upgrade with the master, maybe my rebase fooled me, I will adjust sorry